setup.py

```diff
@@ -12,7 +12,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/sageteamorg/python-sage-imap",
-    projcet_urls={
+    project_urls={
```

Comment:
When you change base code, we must also change setup.py version for next build on pypi

Reply:
@sepehr-akbarzadeh, thank you so much for your valuable feedback! 🙏 I've made the changes based on your suggestions. I hope these updates address the issues effectively. If there are any further adjustments needed, please let me know! 😊